cdap-docs/cdap-apps/source/tracker/index.rst

```diff
@@ -0,0 +1,475 @@
+of datasets, streams, and stream views.
+
+It allows you to perform impact and root-cause analysis, and delivers an audit-trail for
+auditability and compliance. Tracker furnishes access to structured information that
+describes, explains, locates, and makes it easier to retrieve, use, and manage datasets.
+
+Tracker also allows for the storage of metadata where it can be accessed and indexed. This
+allows it to be is easily searched and provides consistent, high-quality metadata.
+
+Tracker's UI shows a graphical visualization of the :ref:`lineage
+<metadata-lineage-lineage>` of an entity. A lineage shows |---| for a specified time range
+|---| all data access of the entity, and details of where that access originated from.
+
+
+**Harvest, Index, and Track Datasets**
+
+- Immediate, timely, and seamless capture of technical, business, and operational metadata,
+  enabling faster and better traceability of all datasets.
+
+- Tracker quickly, reliably, and accurately indexes technical, business, and operational metadata
+  to help locate datasets.
+
+- Through its use of lineage, it lets you understand the impact of changing a dataset on
+  other datasets, processes or queries.
+
+- Tracks the flow of data across enterprise systems and data lakes.
+
+- Provides trusted and complete metadata on datasets, enabling traceability to resolve
+  data issues and to improve data quality.
+
+
+**Supports Standardization, Governance, and Compliance Needs**
+
+- Provide IT with the traceability needed in governing datasets and in applying compliance
+  rules through seamless integration with other extensions.
+
+- Tracker has consistent definitions of metadata-containing information about the data to reconcile
+  differences in terminologies.
+
+- It helps you in the understanding of the lineage of your business-critical data.
+
+
+**Blends Metadata Analytics and Integrations**
+
+- See how your datasets are being created, accessed, and processed.
+
+- Extensible integrations are available with enterprise-grade MDM (master data management)
+  systems such as `Cloudera Navigator <https://www.cloudera.com/products/cloudera-navigator.html>`__ 
+  for centralizing metadata repository and the delivery of complete, accurate, and correct
+  data.
+
+
+Example Use Case
+----------------
+An example use case describes how Tracker was employed in the `data cleansing and validating of
+three billion records <http://customers.cask.co/rs/882-OYR-915/images/tracker-casestudy1.pdf>`__.
+
+
+Tracker Application
+===================
+The Cask Tracker application consists of an application in CDAP with two programs and two datasets:
+
+- ``_Tracker`` application: names begins with an underscore
+- ``AuditLog`` service: it exposes the Tracker audit log as an API
+- ``AuditLogFlow`` flow: subscribes to Kafka audit messages and stores them in the ``AuditLog``	dataset
+- ``AuditLog`` custom dataset: type ``co.cask.tracker.entity.AuditLogTable``
```

Comment:
we are changing these datasets to be _AuditLog and _kafkaOffset and they will be hidden in the CDAP UI